src/ui/utils/LedgerContext.js

```diff
@@ -14,17 +15,21 @@ type LedgerContextValue = {|
 |};
 
 type LedgerProviderProps = {|
-  +initialLedger: Ledger,
+  +ledgerManager: LedgerManager,
   +children: React.Node,
 |};
 
 export const LedgerProvider = ({
   children,
-  initialLedger,
+  ledgerManager,
 }: LedgerProviderProps): React.Node => {
-  const [{ledger}, setLedgerState] = React.useState({ledger: initialLedger});
+  const [{ledger}, setLedgerState] = React.useState({
+    ledger: ledgerManager.ledger,
+  });
 
-  const updateLedger = (ledger: Ledger) => setLedgerState({ledger});
+  // Workaround to trigger UI update in React since ledger object is mutable
+  const updateLedger = (_?: Ledger) =>
```

Comment:
Why keep the param?

Reply:
Ah I think I understand. Because this is only one implementation of theoretically many, and others may need the param.

Reply:
Yea exactly, didn't want to break existing code that was passing in the ledger since we might need it in future implementations.